x-pack/test/security_solution_endpoint_api_int/apis/index.ts

```diff
@@ -24,9 +26,22 @@ export default function endpointAPIIntegrationTests(providerContext: FtrProvider
     const registryUrl = getRegistryUrlFromTestEnv() ?? getRegistryUrlFromIngest();
     log.info(`Package registry URL for tests: ${registryUrl}`);
 
+    const roles = Object.values(ROLE);
     before(async () => {
       await ingestManager.setup();
+
+      // create role/user
+      for (const role of roles) {
```

Comment:
Doing this here in order to create/delete all roles just one time. Let me know what you think about it.